lib/watir/elements/element.rb

```diff
@@ -529,6 +529,16 @@ def reset!
       @element = nil
     end
 
+    #
+    # Allows this instance to respond to editing methods
+    #
+    # @see Watir::UserEditable
+    #
+
+    def make_editable
+      self.extend UserEditable
```

Comment:
Does `#append` work for you? If the element isn't already in edit mode, the value appends to the beginning for me (was using Chrome).

Do you care that `#set!` won't work in this case?